protos/sources/vega/vega.proto

```diff
@@ -1748,29 +1745,27 @@ message VolumeDiscountProgram {
   uint64 version = 1;
   // Unique ID generated from the proposal that created this program.
   string id = 2;
-  // Defined benefit tiers in increasing order. First element will give Tier 1,
-  // second element will give Tier 2, and so on. Determines the level of
-  // benefit a party can expect based on performance criteria.
+  // Defined benefit tiers ordered by increasing discounts.
   repeated VolumeBenefitTier benefit_tiers = 3;
-  // Timestamp as Unix time in seconds, after which when the current epoch
-  // ends, the programs will end and benefits will be disabled.
+  // Timestamp in Unix nanoseconds, after which when the current epoch
```

Comment:
This is definitely nanoseconds, right?

Reply:
Whoops, this one is seconds still, good spot.